### DISCUSSION TITLE
Further Issues with TypeScript example

### DISCUSSION THREAD

**Discussion by Kaszanas** [grpc/grpc-web#1439] [Q&A - Errors / Debugging Help] [answered]
While solving the https://github.com/grpc/grpc-web/issues/1349 with PRs: https://github.com/grpc/grpc-web/pull/1350 and https://github.com/grpc/grpc-web/pull/1352

Entirely new error was discovered, this time on:

```
 => ERROR [prereqs copy-and-build 21/21] RUN /usr/local/bin/buildifier   --mode=check --lint=warn --warnings=all -r ./WORKSPACE javascript net                                                                                             0.5s
------
 > [prereqs copy-and-build 21/21] RUN /usr/local/bin/buildifier   --mode=check --lint=warn --warnings=all -r ./WORKSPACE javascript net:
0.504 ./WORKSPACE # reformat
0.504 javascript/net/grpc/web/generator/BUILD.bazel # reformat
0.504 net/grpc/gateway/examples/echo/BUILD.bazel # reformat
------
failed to solve: process "/bin/sh -c /usr/local/bin/buildifier   --mode=check --lint=warn --warnings=all -r ./WORKSPACE javascript net" did not complete successfully: exit code: 4
```

You may be interested in this @sampajano

**Comment by sampajano** [comment]
Sorry I had mistakenly converted this into a discussion 😅

**Reply by sampajano** [reply, marked as answer]
Currently on my macbook (macbook air m2), the following command is now working :)

```docker-compose up --build node-server envoy ts-client```

**Reply by Kaszanas** [reply]
This was almost a year ago, so unfortunately I no longer have access to the code that was causing this.
It is unlikely that I will be attempting to solve this soon.

I may try to run the example but I cannot guarantee anything @sampajano 

Will see what I can do.

**Reply by sampajano** [reply]
@Kaszanas No worries at all. Thank you for working on it back then. And sorry for the delay in addressing this :) No obligation at all but if you run into anything in the examples do let us know. Thank you :)
